Stonith/docs/3nodes/ipmi_stonith.md

```diff
@@ -0,0 +1,178 @@
+pcs stonith create stonith-c3 fence_ipmilan ipaddr=10.230.241.149 login=ADMIN passwd=adminBMC! delay=30 pcmk_host_list=node3 pcmk_host_check=static-list lanplus=true auth=PASSWORD power_timeout=40 verbose=true op monitor interval=10s meta failure-timeout=15s
+
+*** Added location constraint to avoid action on same node
+ 
+pcs constraint location stonith-c1 avoids node1
+pcs constraint location stonith-c2 avoids node2
+pcs constraint location stonith-c3 avoids node3
+
+*** Clone stonith resources
+
+pcs resource clone stonith-c1
+pcs resource clone stonith-c2
+pcs resource clone stonith-c3
+
+[root@smc40-m09 cluster]# pcs status
+Cluster name: test_cluster
+Stack: corosync
+Current DC: node2 (version 1.1.23-1.el7-9acf116022) - partition with quorum
+Last updated: Sun Feb 14 23:01:59 2021
+Last change: Sun Feb 14 22:54:17 2021 by root via cibadmin on node1
+
+3 nodes configured
+12 resource instances configured
+
+Online: [ node1 node2 ]
+OFFLINE: [ node3 ]
+
+Full list of resources:
+
+ Clone Set: dummy-clone [dummy]
+     Started: [ node1 node2 node3 ]
+ Clone Set: stonith-c1-clone [stonith-c1]
+     Started: [ node2 node3 ]
+     Stopped: [ node1 ]
+ Clone Set: stonith-c2-clone [stonith-c2]
+     Started: [ node1 node3 ]
+     Stopped: [ node2 ]
+ Clone Set: stonith-c3-clone [stonith-c3]
+     Started: [ node1 node2 ]
+     Stopped: [ node3 ]
+   
+Daemon Status:
+  corosync: active/enabled
+  pacemaker: active/enabled
+  pcsd: active/enabled  
+
+
+*** Check stonith device status
+
+[root@smc33-m09 ~]# fence_ipmilan -a 10.230.241.149 -l ADMIN -p adminBMC! -o status
+Status: ON
+[root@smc33-m09 ~]# fence_ipmilan -a 10.230.244.123 -l ADMIN -p adminBMC! -o status
+Status: ON
+[root@smc33-m09 ~]# fence_ipmilan -a 10.230.244.126 -l ADMIN -p adminBMC! -o status
+Status: ON
+
+==============================
+Scenario 1
+==============================
+*** Did manual fence testing by taking ip link down:
+ip link set down enp175s0f1
+
+After the above command execution it has shown the output like below:
+
+[root@smc40-m09 cluster]# pcs status
+Cluster name: test_cluster
+Stack: corosync
+Current DC: node2 (version 1.1.23-1.el7-9acf116022) - partition with quorum
+Last updated: Sun Feb 14 23:01:59 2021
+Last change: Sun Feb 14 22:54:17 2021 by root via cibadmin on node1
+
+3 nodes configured
+12 resource instances configured
+
+Online: [ node1 node2 ]
+OFFLINE: [ node3 ]
+
+Full list of resources:
+
+ Clone Set: dummy-clone [dummy]
+     Started: [ node1 node2 ]
+     Stopped: [ node3 ]
+ Clone Set: stonith-c1-clone [stonith-c1]
+     Started: [ node2 ]
+     Stopped: [ node1 node3 ]
+ Clone Set: stonith-c2-clone [stonith-c2]
+     Started: [ node1 ]
+     Stopped: [ node2 node3 ]
+ Clone Set: stonith-c3-clone [stonith-c3]
+     Started: [ node1 node2 ]
+     Stopped: [ node3 ]
+
+Daemon Status:
+  corosync: active/enabled
+  pacemaker: active/enabled
+  pcsd: active/enabled
+
+Means node3 is stopped and we are good with our POC.
```

Comment:
So node3 was not reachable by node1 and node2.
Did fencing was attempted by both the nodes, node1 and node2?
Let's node1 was able to fence node3. What will be the status of stonith-c3-clone on node2 trying to fence node1?

Reply:
@ajaysrivas I have added minute details in result section in the file.

Reply:
@shindeam The logs are included in the result but actual question is not answered. Anyone looking at the results, needs to analyze logs. You should conclude the behavior.
Also, where is the measure of n/w trafiic?

Reply:
Ok agree, added result in words.

Reply:
Please check this PR - https://github.com/Seagate/cortx-experiments/pull/8/files
nload tool was used by Andrei for measuring n/w traffic.

Reply:
yes used same & added the network measures for all nodes. updated in document & ticket comment too